python/pyspark/pandas/utils.py

```diff
@@ -466,7 +471,11 @@ def is_testing() -> bool:
 
 
 def default_session() -> SparkSession:
-    spark = SparkSession.getActiveSession()
+    if not is_remote():
+        spark = SparkSession.getActiveSession()
+    else:
+        # TODO: Implement `getActiveSession` for SparkConnect.
+        spark = None
```

Comment:
there is a workaround for now

```suggestion
        from pyspark.sql.connect.session import _active_spark_session

        spark = _active_spark_session  # type: ignore[assignment]
```

Reply:
Cool. Thanks!